lib/curl_sha512_256.c

```diff
@@ -548,8 +548,8 @@ MHDx_sha512_256_transform(curl_uint64_t H[SHA512_256_HASH_SIZE_WORDS],
        'wt' calculation may change other data as well every time when
        used. */
 #define SHA2STEP64(vA,vB,vC,vD,vE,vF,vG,vH,kt,wt) do {                  \
-      (vD) += ((vH) += SIG1 ((vE)) + Ch ((vE),(vF),(vG)) + (kt) + (wt)); \
-      (vH) += SIG0 ((vA)) + Maj ((vA),(vB),(vC)); } while (0)
+      (vD) += ((vH) += SIG1 ((vE)) + Sha512_Ch((vE),(vF),(vG)) + (kt) + (wt)); \
+      (vH) += SIG0 ((vA)) + Sha512_Maj((vA),(vB),(vC)); } while (0)
```

Comment:
```suggestion
      (vD) += ((vH) += SIG1((vE)) + Sha512_Ch((vE),(vF),(vG)) + (kt) + (wt)); \
      (vH) += SIG0((vA)) + Sha512_Maj((vA),(vB),(vC)); } while (0)
```
Still need to find at least one character to fit `checksrc`.
Maybe `SIG1(vE)`? `SIG1()` (and `SIG0()`) already put the argument into parenthesis.

Reply:
can also rename `Sha512_Maj` to `S512_Maj`

Reply:
I'd vote for a consistent prefix. Perhaps just splitting the line would work instead?